gordo_components/cli/cli.py

```diff
@@ -231,12 +246,21 @@ def build(
 
             for score in all_scores:
                 print(score)
+
+        # If enabled, configure remote logging to AzureML, otherwise logs locally
+        workspace_kwargs = get_workspace_kwargs() if enable_remote_logging else {}
+        service_principal_kwargs = get_spauth_kwargs() if enable_remote_logging else {}
+        with mlflow_context(
+            name, builder.cache_key, workspace_kwargs, service_principal_kwargs
+        ) as (mlflow_client, run_id):
+            log_metadata(mlflow_client, run_id, metadata)
+
     except Exception as e:
         exit_code = EXCEPTION_TO_EXITCODE.get(e.__class__, 1)
         traceback.print_exc()
         sys.exit(exit_code)
-    else:
-        return 0
+
```

Comment:
Small thing, but seems rebasing removed the `else` here.

Reply:
That was just some clean up, now it is like this
```python
try:
    exit_code <- build()
exception:
    sys.exit(exit_code)
return 0
```

Reply:
https://github.com/equinor/gordo-components/blob/4098901a0da5222a5f27b0338865e089610ac735/gordo_components/cli/cli.py#L238

Reply:
Ah... I fixed that already locally and apparently haven't pushed that. That said, I see now the point of the `try/except/else` pattern here, so changing it back :+1: